jscomp/ext/cli_args.ml

```diff
@@ -0,0 +1,41 @@
+(* Copyright (C) 2021- Authors of Melange
```

Comment:
Let's call this `ext_cli_args.ml` and add a `.mli` file, please!

Reply:
Done!